CMakeLists.txt

```diff
@@ -650,8 +652,27 @@ if (BUILD_TESTING)
                     -Wno-unknown-pragmas -Wformat-security -Wno-missing-braces -fvisibility=hidden -Wno-unreachable-code
                     -Wno-unused-but-set-variable)
         endforeach(benchmark)
-
     endif()
+
+    find_package (Python3 COMPONENTS Interpreter Development)
+    file(GLOB integv2_test_files "${PROJECT_SOURCE_DIR}/tests/integrationv2/test_*.py")
+    foreach(test_file_path ${integv2_test_files})
+        get_filename_component(test_filename ${test_file_path} NAME_WE)
+        string(REGEX REPLACE "^test_" "integrationv2_" test_target ${test_filename})
+        add_test(NAME ${test_target}
+            COMMAND
+            ${CMAKE_COMMAND} -E env
+            DYLD_LIBRARY_PATH=${PROJECT_SOURCE_DIR}/libcrypto-root/lib:$ENV{DYLD_LIBRARY_PATH}
+            LD_LIBRARY_PATH=${PROJECT_SOURCE_DIR}/libcrypto-root/lib:${PROJECT_SOURCE_DIR}/test-deps/openssl-1.1.1/lib:${PROJECT_SOURCE_DIR}/test-deps/gnutls37/nettle/lib:$ENV{LD_LIBRARY_PATH}
+            PATH=${PROJECT_SOURCE_DIR}/bin:${PROJECT_SOURCE_DIR}/test-deps/openssl-1.1.1/bin:${PROJECT_SOURCE_DIR}/test-deps/gnutls37/bin:$ENV{PATH}
+            PYTHONNOUSERSITE=1
+            S2N_INTEG_TEST=1
+            TOX_TEST_NAME=${test_file_path}
+            ${Python3_EXECUTABLE} -m tox
+            WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/tests/integrationv2)
+        set_property(TEST ${test_target} PROPERTY LABELS "integrationv2")
```

Comment:
You can actually write anything that regex matches the labels. So:

`test -L in` will run all the integrationv2 tests.

`test -L u` will run all the unit tests. Hilariously `tests -L ni` will also run the u*ni*t tests.

Reply:
wow that is kind of bizarre. I suppose its nice? But unintuitive. We will need to write some good docs when we remove make, because that is really weird.